Assets/Mirror/Editor/Weaver/CompilationFinishedHook.cs

```diff
@@ -20,7 +20,10 @@ public static class CompilationFinishedHook
 
         public static bool WeaverEnabled { get; set; } // controls whether we weave any assemblies when CompilationPipeline delegates are invoked
         public static bool UnityLogEnabled = true; // controls weather Weaver errors are reported direct to the Unity console (tests enable this)
-        public static bool WeaveFailed { get; private set; } // holds the result status of our latest Weave operation
+
+        // holds the result status of our latest Weave operation
+        // NOTE: WeaveFailed is critical to unit tests, but isn't used for anything else. 
+        public static bool WeaveFailed { get; private set; } 
```

Comment:
can you add this comment in a separate commit first please?
no need to do too much at once here

Reply:
Done